plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/PostgreMessages.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ * Copyright (C) 2017 Liu, Yuanyuan (liuyuanyuan@highgo.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.postgresql;
+
+import org.eclipse.osgi.util.NLS;
+
+public class PostgreMessages extends NLS {
```

Comment:
We already have PostgreMessages https://github.com/dbeaver/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/PostgreMessages.java

Reply:
Yes, but they are in UI plugin. Is there a way to use them from another plugin? I could not do this, so created new file

Reply:
@LonwoLonwo , finally what should I do?

Reply:
I tested your code and found that the reason for this https://github.com/dbeaver/dbeaver/pull/18588#issuecomment-1386803374 is the following error. Actually such errors happens not only in this case, but even when trying to create new connection for postgresql. It occurred because `PostgreMessages` class with localization fields and bundle name `"org.jkiss.dbeaver.ext.postgresql.PostgresResources"` are absolutely the same for ui bundle and non-ui bundle where you added them. Probably `NLS.initializeMessages(BUNDLE_NAME, PostgreMessages.class);` should have unique arguments otherwise previous initialization will be overwritten.

So I suggest to rename the class or/and resource bundle. Maybe renaming .properties files would be enough.
```
java.lang.NoSuchFieldError: source_view_show_header_label
	at org.jkiss.dbeaver.ext.postgresql.ui.editors.PostgreSourceViewEditor.contributeEditorCommands(PostgreSourceViewEditor.java:90)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditorNested$EditorPageControl.fillCustomActions(SQLEditorNested.java:415)
	at org.jkiss.dbeaver.ui.controls.ProgressPageControl.fillCustomActions(ProgressPageControl.java:259)
	at org.jkiss.dbeaver.ui.controls.ObjectEditorPageControl.fillCustomActions(ObjectEditorPageControl.java:121)
	at org.jkiss.dbeaver.ui.editors.entity.properties.ObjectPropertiesEditor$1.fillCustomActions(ObjectPropertiesEditor.java:116)
	at org.jkiss.dbeaver.ui.controls.ProgressPageControl.hideControls(ProgressPageControl.java:304)
	at org.jkiss.dbeaver.ui.controls.ProgressPageControl.setChildControl(ProgressPageControl.java:173)
	at org.jkiss.dbeaver.ui.controls.ProgressPageControl.substituteProgressPanel(ProgressPageControl.java:139)
	at org.jkiss.dbeaver.ui.controls.ProgressPageControl.createOrSubstituteProgressPanel(ProgressPageControl.java:146)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditorNested.createPartControl(SQLEditorNested.java:143)
	at org.jkiss.dbeaver.ui.editors.entity.properties.TabbedFolderPageEditor.createControl(TabbedFolderPageEditor.java:98)
	at org.jkiss.dbeaver.ui.controls.folders.TabbedFolderComposite$FolderPane.onFolderSwitch(TabbedFolderComposite.java:153)
	at org.jkiss.dbeaver.ui.controls.folders.TabbedFolderComposite$FolderPane$2.widgetSelected(TabbedFolderComposite.java:136)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:252)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
```

Reply:
@E1izabeth , I agree with renaming, can you suggest proper class name? PGMessages?

Reply:
I'd call it PostgreSQLMessages